src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --background-color: #fff;
+  --text-color: #000;
+  --headerlink-color: #00acdc;
```

Comment:
Nice use of root to declare css vars, maybe just header-link-color would be cleaner but it's like a really minor detail

Reply:
Great, I'll make sure to pay attention to it!